R/serialization.R

```diff
@@ -0,0 +1,41 @@
+# This file will be autogenerated.
+
+#' Serialize a model to the given filename.
+#'
+#' @title Serialize.
+#' @param filename Input filename.
+#' @param model_in Input model.
```

Comment:
Add here:

```r
#' @export
#' @rdname mlpack-serialization
```

This will create a help documentation entry that can be used for both serialization and _unserialization_.

Reply:
Done.